pg/Dockerfile

```diff
@@ -1,4 +1,18 @@
-FROM postgres:14.3-alpine
+FROM postgres:14.3-alpine as dataloader
+
 
 COPY input_data /input_data/
+
 COPY scripts/init.sql /docker-entrypoint-initdb.d/
+ENV POSTGRES_USER=postgres
+ENV POSTGRES_PASSWORD=change-me
+ENV PGDATA=/data
+
+# Make sure postrgres is not actually started after data import
+RUN ["sed", "-i", "s/exec \"$@\"/echo \"skipping...\"/", "/usr/local/bin/docker-entrypoint.sh"]
+RUN ["/usr/local/bin/docker-entrypoint.sh", "postgres"]
+
+FROM postgres:14.3-alpine
```

Comment:
use multistage build, where the first step reads and process all data and the second and steals/copies the db data and then runs as a standard postgres container